pages/linux/debman.md

```diff
@@ -1,6 +1,7 @@
 # debman
 
 > Read man pages from uninstalled packages.
+> More information: <https://packages.debian.org/buster/debian-goodies>.
```

Comment:
```suggestion
> More information: <https://manpages.debian.org/latest/debian-goodies/debman.1.html>.
```
We generally try to use man pages. (Also, when using `dabian.org` links we sholud use `latest` and remove the `en` locale.)